sdk/keyvault/azure-security-keyvault/tests/test_secrets_client.py

```diff
@@ -99,16 +99,14 @@ def _update_secret(secret):
         if self.is_live:
             # wait to ensure the secret's update time won't equal its creation time
             time.sleep(1)
+
```

Comment:
_update_secret can be updated to check for other values passed for updation i.e `content_type` `expires` ..
Also, we can avoid returning the secret_bundle for `_update_secret` we don't really do much with it.